lib/core_ext/string.rb

```diff
@@ -0,0 +1,7 @@
+# License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
+
+class String
+  def is_int?
+    !!(self =~ /\A\d+\z/)
+  end
+end
```

Comment:
Why this design decision to create this core_exts folder? I don't think it's bad, in fact, I can see advantages to doing it this way. How did the decision come about?

Reply:
I did some research on how to do it, I saw that some people created just an `ext` folder and someone suggested creating as `core_ext`. I checked how it is being done on Rails and that's the name of the folder that holds each ActiveSupport type classes: https://github.com/rails/rails/tree/main/activesupport/lib/active_support/core_ext

I could also follow the same pattern and create a folder called `string` and a file called `string.rb`, and a class called `types.rb` that checks if the string is compatible with some type. For example:

```
├── lib
│   ├── core_ext
│   │   ├── string
│   │   │   ├── types.rb
│   │   ├── string.rb
```

Reply:
I think the design you have now makes sense. Nice job!